src/material/menu/menu-trigger.ts

```diff
@@ -174,25 +180,23 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
               private _element: ElementRef<HTMLElement>,
               private _viewContainerRef: ViewContainerRef,
               @Inject(MAT_MENU_SCROLL_STRATEGY) scrollStrategy: any,
-              // `MatMenu` is always used in combination with a `MatMenuTrigger`.
-              // tslint:disable-next-line: lightweight-tokens
-              @Optional() private _parentMenu: MatMenu,
+              @Inject(MAT_MENU_PANEL) @Optional() parentMenu: MatMenuPanel,
```

Comment:
Isn't this a breaking constructor change?

Reply:
I don't think so since `MatMenu` implements `MatMenuPanel`.

Reply:
Ah, yeah I think it's ok then